sections/conclusion/main.tex

```diff
@@ -1,58 +1,69 @@
+    serve as a parking space for the ambulances
+    \item Analytic performance measure formulas for the queueing model
+    \item A 3-player game theoretic model between the EMS and two EDs
+    \item Numerical experiments showing emergent behaviour of gaming between
+    EDs and the EMS.
+\end{itemize}
 
-This study explores a generic 3-player game theoretic model between the 
-decision makers of two queueing systems and a service that distributes 
+This study explores a generic 3-player game theoretic model between the
+decision makers of two queueing systems and a service that distributes
 individuals to these two systems (section \ref{sec:model_overview}).
-It also describes the construction of the underlying queueing theoretic model 
-that has a tandem buffer and a single service centre (section 
+It also describes the construction of the underlying queueing theoretic model
+that has a tandem buffer and a single service centre (section
 \ref{sec:queueing_model}).
-Furthermore, the formulas for the performance measures of the queueing model 
-are also derived (sections \ref{sec:waiting_time}, \ref{sec:blocking_time}, 
-\ref{sec:proportion_within_target}). 
+Furthermore, the formulas for the performance measures of the queueing model
+are also derived (sections \ref{sec:waiting_time}, \ref{sec:blocking_time},
+\ref{sec:proportion_within_target}).
 This novel queuing model is the first contribution of the paper.
 The game theoretic model is then applied to a healthcare scenario by looking at
 the interface between the EDs and the EMS.
-The inefficiencies that emerge from the perspective of the EMS were explored 
+The inefficiencies that emerge from the perspective of the EMS were explored
 along with ways to apply some incentive mechanisms to improve them.
 The key findings from this paper that were observed when playing the game
 between two EDs and the EMS are:
 \begin{itemize}
     \item Inefficiencies can be learned and emerge naturally
     \item Targeted incentivisation of behaviours can help escape inefficiencies
 \end{itemize}
-% TODO: Make sure this is consistent with `intro' and `lit review' sections
-The former relates to the results of asymmetric replicator dynamics that showed 
-that inefficient scenarios can arise by playing the game while the latter 
-implies that the learned inefficiencies can be escaped by carefully applying 
+The former relates to the results of asymmetric replicator dynamics that showed
+that inefficient scenarios can arise by playing the game while the latter
+implies that the learned inefficiencies can be escaped by carefully applying
 certain incentives to the players.
 This applied game theoretic model is the second main contribution of this paper.
 
-The model presented here assumes the presence of two players that can receive 
-individuals. 
-However, in a realistic healthcare scenario an ambulance may have to decide 
+The model presented here assumes the presence of two players that can receive
```

Comment:
```suggestion
The model presented here assumes the presence of just two players that can receive
```

Reply:
Alex suggests that `just` may be insensitive here. Will make it `only two players`